internal/pkg/externalplugins/lgtm/lgtm.go

```diff
@@ -439,8 +441,11 @@ The full list of commands accepted by this bot can be found [here]({{ .commandHe
 
 Reviewer can indicate their review by writing `+"`/lgtm`"+` in a comment.
 Reviewer can cancel approval by writing `+"`/lgtm cancel`"+` in a comment.
-</details>`, "message", map[string]interface{}{"reviewers": reviewedReviewers, "commandHelpLink": commandHelpLink,
-		"prProcessLink": prProcessLink, "ownersLink": ownersLink, "org": org, "repo": repo})
+</details>
+
+<!--{{ .reviewNotificationIdentifier }}-->
+`, "message", map[string]interface{}{"reviewers": reviewedReviewers, "commandHelpLink": commandHelpLink,
```

Comment:
It might be better to store the map in a variable.

Reply:
How about know?

Reply:
Better than before.